src/fallbackRest.ts

```diff
@@ -61,6 +61,10 @@ export function encodeRequest(
     rpc.parsedOptions,
     rpc.resolvedRequestType!.fields
   );
+  if (transcoded instanceof GoogleError) {
```

Comment:
Here I think it's pretty much the same as putting the call to `transcode()` into `try ... catch`, then we won't need to change its return value. It may or may not be cleaner.